sound/soc/sof/sof-priv.h

```diff
@@ -468,6 +468,8 @@ int snd_sof_runtime_resume(struct device *dev);
 int snd_sof_runtime_idle(struct device *dev);
 int snd_sof_resume(struct device *dev);
 int snd_sof_suspend(struct device *dev);
+int snd_sof_set_dsp_state(struct snd_sof_dev *sdev,
```

Comment:
please be careful with naming, you use set_state, set_dsp_state, but what you are doing is set_power_state()

Reply:
OK, let me change it to set_power_state